src/Model/Item.php

```diff
@@ -2333,16 +2334,15 @@ private static function autoReshare(array $item)
 			return;
 		}
 
-		$cdata = Contact::getPublicAndUserContactID($item['author-id'], $item['uid']);
```

Comment:
Have you removed that line intentionally? Directly below you use the `$cdata` variable.

Reply:
Look at the original code, the $cdata variable is initialized twice the same way. I removed the duplicate and dded a new check, which the diff then superposes.

Reply:
Oookay, that looks weird. For whatever reason some lines are duplicated.